### ISSUE TITLE
determining alignment score

### ISSUE THREAD

**Issue by ekg** [rvaser/spoa#39]
How would you recommend computing the alignment score from an alignment object.

In my application, I'd like to test this score to estimate if the alignment should be made in the forward or reverse orientation.

**Comment by rvaser** [comment]
It is a bit tricky to get the score after the alignment. I'll add an optional argument to the alignment function for the score.

**Comment by ekg** [comment]
Any suggestions on this? I could use it for an ongoing project. If you can describe it quickly perhaps I can implement it.

**Comment by rvaser** [comment]
Hi Erik,
I am really sorry for this delay. I have pushed your request to a new branch https://github.com/rvaser/spoa/tree/score. Signature of the alignment function has been changed to:
```cpp
Alignment align(const std::string& sequence, const std::unique_ptr<Graph>& graph, std::int32_t* score = nullptr);
```
If you need anything else, let me know.

Best regards,
Robert

**Comment by ekg** [comment]
Wow! Thank you!!

**Comment by rvaser** [comment]
Included from `v4.0.0`
